### PR TITLE
Adding accumulation chart sample

### PR DESCRIPTION
**Pull request by vinoth-Kumar-S**
Adding accumulation chart sample

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
-# how-to-create-react-accumulation-charts-of-syncfusion
-A quick-start react project that depicts the creation of Syncfusion Accumulation Charts with appropriate title, legend, data labels, and tooltip.
+# How to Create React Accumulation Charts of Syncfusion
+
+A quick-start project that allows you to create an accumulation charts of Syncfusion in the react application. This project also contains a code snippet to add a title, legend, data label, and tooltip to the pie chart.
```
Comment (UmaMaheswariChandrabose) [inline README.md]: Revisit the sentence, as I can notice grammatical errors. Also, at last you just mentioned pie chart alone.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+
+Documentation: https://ej2.syncfusion.com/react/documentation/accumulation-chart/getting-started/
+
+Online examples: https://ej2.syncfusion.com/react/demos/#/material/chart/default-pie
+
+## Project prerequisites
+
+Make sure that you have the latest versions of NodeJS and visual studio code in your machine before starting to work on this project.
```
Comment (UmaMaheswariChandrabose) [inline README.md]: Visual Studio Code --> Follow this casing

File: README.md (hunk 1 above)
Comment (vinoth-Kumar-S) [reply README.md]: Done the changes

File: README.md (hunk 2 above)
Comment (vinoth-Kumar-S) [reply README.md]: Done the changes

**Pull request merged**